systemtest/intake_async_test.go

```diff
@@ -56,17 +56,17 @@ func TestIntakeAsync(t *testing.T) {
 		systemtest.CleanupElasticsearch(t)
 		srv := apmservertest.NewServerTB(t)
 
-		systemtest.SendBackendEventsAsyncPayload(t, srv.URL, `../testdata/intake-v2/errors.ndjson`)
+		systemtest.SendBackendEventsPayload(t, srv.URL, `../testdata/intake-v2/errors.ndjson`)
```

Comment:
Let's revert this change, there's no need to process these synchronously since the line below asserts that the documents have been indexed to Elasticsearch, at which point, all the requests have been handled.
```suggestion
		systemtest.SendBackendEventsAsyncPayload(t, srv.URL, `../testdata/intake-v2/errors.ndjson`)
```

Reply:
Good catch!